docs/examples/image_processing/index.rst

```diff
@@ -0,0 +1,15 @@
+Image Processing
+================
+
+.. toctree::
+   :maxdepth: 2
+
+   data_loading/index
+   augmentation_gallery.ipynb
+   brightness_contrast_example.ipynb
+   color_space_conversion.ipynb
+   decoder_examples.ipynb
+   hsv_example.ipynb
+   warp_3D.ipynb
```

Comment:
How about a separate sub-category for 3D?

Reply:
Since there's only 1 example for it, I'd pass right now. In future sure, but now it'll be just empty...